cmd/cli/plugin-admin/codegen/main.go

```diff
@@ -7,16 +7,19 @@ import (
 	"github.com/aunum/log"
 
 	cliapi "github.com/vmware-tanzu/tanzu-framework/cli/runtime/apis/cli/v1alpha1"
-	"github.com/vmware-tanzu/tanzu-framework/cli/runtime/buildinfo"
+	pluginrtbuildinfo "github.com/vmware-tanzu/tanzu-framework/cli/runtime/buildinfo"
 	"github.com/vmware-tanzu/tanzu-framework/cli/runtime/plugin"
 )
 
 var descriptor = cliapi.PluginDescriptor{
 	Name:        "codegen",
 	Description: "Tanzu code generation tool",
 	Group:       cliapi.AdminCmdGroup,
-	Version:     buildinfo.Version,
-	BuildSHA:    buildinfo.SHA,
+	// TODO: When plugins have their own buildInfo, we need to update "Version" and "BuildSHA"
+	// 		to plugin own versions instead of plugin runtime version
```

Comment:
When plugins will use their own Makefile, they will inject their own versions into these build variables.
You can actually see this already in `cmd/cli/plugin-admin/builder/template/plugintemplates/Makefile.tmpl`.
So I think we can remove the TODO